x-pack/plugins/encrypted_saved_objects/README.md

```diff
@@ -68,7 +68,19 @@ router.get(
 ...
 ```
 
-5. To retrieve Saved Object with decrypted content use the dedicated `getDecryptedAsInternalUser` API method. 
+5. Instancicate an EncryptedSavedObjects client so that you can interact with Saved Objects whose content has been encrypted.
```

Comment:
__typo:__
```suggestion
5. Instantiate an EncryptedSavedObjects client so that you can interact with Saved Objects whose content has been encrypted.
```